src/assets/scss/_about-me.scss

```diff
@@ -0,0 +1,80 @@
+//About me section
+
+.about-me {
+    background-color: $background-section-color;
+    height: 38rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    width: 100%;
+    
+    
+
+    &--hi-slider {
+      display: flex;
+      font-family: $sans-serif;
+      font-size: 15rem;
+      font-weight: 300;
+      color: #9EC8C8;
+      opacity: 0.7;
+      position: relative;
+      z-index: 1;
+      overflow: hidden;
+      width: inherit;
+      animation: --hi-slider 8s linear infinite;
+
+      @keyframes --hi-slider {
```

Comment:
mnie uczyli (:D), że jak na coś się powołujesz to powinno to być nad wywołaniem. czyt. to powinno być nad &--hi-slider, a najlepiej na samej górze :)